aries-site/src/data/wcag/components.json

```diff
@@ -861,6 +861,518 @@
         "label": "Status Messages",
         "status": "not-applicable"
       }
+    ],
+    "textinput": [
+      {
+        "rule": "1.1.1",
+        "label": "Non-text content",
+        "status": "conditional",
+        "notes": "If an icon is provided, an aria-label can be added to the icon."
+      },
+      {
+        "rule": "1.2.1",
+        "label": "Audio-only and Video-only (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.2",
+        "label": "Captions (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.3",
+        "label": "Audio Description or Media Alternative (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.4",
+        "label": "Captions (Live)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.5",
+        "label": "Audio Description (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.6",
+        "label": "Sign Language (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.7",
+        "label": "Extended Audio Description (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.8",
+        "label": "Media Alternative (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.9",
+        "label": "Audio-only (Live)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.1",
+        "label": "Info and Relationships",
+        "status": "conditional",
+        "notes": "An aria attribute can be added."
+      },
+      {
+        "rule": "1.3.2",
+        "label": "Meaningful Sequence",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.3",
+        "label": "Sensory Characteristics",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.4",
+        "label": "Orientation",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.5",
+        "label": "Identify Input Purpose",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.6",
+        "label": "Identify Purpose",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.1",
+        "label": "Use of Color",
+        "status": "passed",
```

Comment:
I feel like this is not passing because we distinguish the 'active' and 'selected' element in the suggestions drop down with the grey background color or the light green background color for selected, there is not any other visual indication of the 'active' state that is meeting contrast requirements